<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>brewNoteWidget</class>
 <widget class="QWidget" name="brewNoteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>brewNote</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/edit.svg</normaloff>:/images/edit.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>626</width>
        <height>438</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolBox" name="toolBox_brewnote">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>252</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>252</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="mouseTracking">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page_preboil">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>242</width>
              <height>349</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Preboil</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout_2">
             <item row="0" column="0">
              <widget class="SmartLabel" name="label_Sg">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string>SG</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>lineEdit_Sg</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="SmartLineEdit" name="lineEdit_Sg">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Preboil gravity</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="SmartLabel" name="label_volIntoBk">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string>Volume</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>lineEdit_volIntoBk</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="SmartLineEdit" name="lineEdit_volIntoBk">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Volume of wort collected</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="SmartLabel" name="label_strikeTemp">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string>Strike Temp</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>lineEdit_strikeTemp</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="SmartLineEdit" name="lineEdit_strikeTemp">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Temperature of strike water before dough in</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="SmartLabel" name="label_mashFinTemp">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string>Final Temp</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>lineEdit_mashFinTemp</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="SmartLineEdit" name="lineEdit_mashFinTemp">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Temperature of mash before mash out</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_postboil">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>242</width>
              <height>349</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Postboil</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="SmartLabel" name="label_Og">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string>OG</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="buddy">
                <cstring>lineEdit_Og</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="SmartLineEdit" name="lineEdit_Og">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Post boil gravity</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="SmartLabel" name="label_postBoilVol">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string>Postboil Volume</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>lineEdit_postBoilVol</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="SmartLineEdit" name="lineEdit_postBoilVol">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Volume of wort in BK after boil</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="SmartLabel" name="label_volIntoFerm">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="toolTip">
                <string>Volume of wort transferred to fermenter</string>
               </property>
               <property name="text">
                <string>Volume into fermenter</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>lineEdit_volIntoFerm</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="SmartLineEdit" name="lineEdit_volIntoFerm">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="SmartLabel" name="label_pitchTemp">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string> Pitch Temp</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="buddy">
                <cstring>lineEdit_pitchTemp</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="SmartLineEdit" name="lineEdit_pitchTemp">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Temperature of wort when yeast is pitched</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_postferment">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>242</width>
              <height>349</height>
             </rect>
            </property>
            <attribute name="label">
             <string>Postferment</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout_5">
             <item row="0" column="0">
              <widget class="SmartLabel" name="label_Fg">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string notr="true">FG</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_Fg</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="SmartLineEdit" name="lineEdit_Fg">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Final gravity</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="SmartLabel" name="label_finalVolume">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string notr="true">Volume</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_finalVolume</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="SmartLineEdit" name="lineEdit_finalVolume">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>100</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="toolTip">
                <string>Volume of beer into serving keg/bottles</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="SmartLabel" name="label_fermentDate">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="text">
                <string notr="true">Date</string>
               </property>
               <property name="buddy">
                <cstring>lineEdit_fermentDate</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QDateTimeEdit" name="lineEdit_fermentDate">
               <property name="date">
                <date>
                 <year>2014</year>
                 <month>10</month>
                 <day>8</day>
                </date>
               </property>
               <property name="displayFormat">
                <string>yyyy-dd-MM</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::ExpandingFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="label_effInfoBk">
               <property name="statusTip">
                <string>Percent efficiency into boil kettle</string>
               </property>
               <property name="text">
                <string>Efficiency into boil kettle</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="SmartDigitWidget" name="lcdnumber_effBK">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="SmartLabel" name="label_projectedOg">
               <property name="contextMenuPolicy">
                <enum>Qt::CustomContextMenu</enum>
               </property>
               <property name="statusTip">
                <string>Expected OG, based on measured FG</string>
               </property>
               <property name="text">
                <string>Projected OG</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="SmartDigitWidget" name="lcdnumber_projectedOG">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_brewHouseEff">
               <property name="statusTip">
                <string>Brewhouse efficiency</string>
               </property>
               <property name="text">
                <string>Brewhouse efficiency</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="SmartDigitWidget" name="lcdnumber_brewhouseEff">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_projectedAbv">
               <property name="statusTip">
                <string>Expected ABV based on recipe OG</string>
               </property>
               <property name="text">
                <string>Projected ABV</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="SmartDigitWidget" name="lcdnumber_projABV">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_Abv">
               <property name="statusTip">
                <string>ABV based on user-reported FG</string>
               </property>
               <property name="text">
                <string>ABV</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="SmartDigitWidget" name="lcdnumber_abv">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_yeastProjAtten">
               <property name="statusTip">
                <string>Yeast attenuation based on yeast specified in recipe</string>
               </property>
               <property name="text">
                <string>Projected yeast attenuation</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="SmartDigitWidget" name="lcdnumber_projAtten">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_yeastAtten">
               <property name="statusTip">
                <string>Yeast attentuation based on user-reported OG and FG</string>
               </property>
               <property name="text">
                <string>Measured yeast attenuation</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="SmartDigitWidget" name="lcdnumber_atten">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Notes</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="BtTextEdit" name="btTextEdit_brewNotes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="mouseTracking">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <customwidgets>
  <customwidget>
   <class>SmartLabel</class>
   <extends>QLabel</extends>
   <header>widgets/SmartLabel.h</header>
   <slots>
    <signal>changedSystemOfMeasurementOrScale(PreviousScaleInfo)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>BtTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">BtTextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>SmartLineEdit</class>
   <extends>QLineEdit</extends>
   <header>widgets/SmartLineEdit.h</header>
   <slots>
    <slot>lineChanged(PreviousScaleInfo)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>SmartDigitWidget</class>
   <extends>QLabel</extends>
   <header>widgets/SmartDigitWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
